src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -220,4 +221,21 @@ describe('AEM Forms OOTB Content Tests', () => {
 
     });
 
+    describe('Theme testing', () => {
+        before(() => {
+            browser.url(config.aem.author.base_url);
+            browser.AEMLogin(config.aem.author.username, config.aem.author.password);
+        });
+
+        const getThemePath = (theme) => '/content/dam/formsanddocuments-themes/${appId}/'+theme+'/jcr:content?wcmmode=disabled';
+        const themes = ['beryl', 'ultramarine', 'urbane', 'tranquil', 'canvas-3-0'];
+        themes.forEach((theme) => {
+            it(theme, () => {
+                browser.url(getThemePath(theme));
+                const isMatched = browser.call(() => browser.matchScreenshot('Theme preview', { errorThreshold: .2 }));
+                expect(isMatched, 'Screenshot did not match').true;
```

Comment:
Can we improve this message by mentioning the theme page in the msg ?

Reply:
done